react/components/AutoComplete/AutoComplete.jsx

```diff
@@ -83,6 +85,19 @@ const AutoComplete = ({
     onChange,
     ...rest
 }) => {
+    if (disabled) {
+        return (
+            <ReadOnlyTextField
+                {...rest}
```

Comment:
if we want user to have an ability to overwrite default props then it is ideal to spread this "rest" at the end.